examples/azure/build-and-deploy-multiple-artifacts.yml

```diff
@@ -0,0 +1,184 @@
+#
+# The example shows how to build, upload and deploy Mender Artifacts for two different devices.
+#
+# 'MENDER_SERVER_ACCESS_TOKEN' secret is required to be set in Azure DevOps pipeline settings.
+#
+
+trigger:
+- main
+
+
+variables:
+- name: MENDER_SERVER_URL # Mender server URL
+  value: https://hosted.mender.io
+- name: MENDER_ARTIFACT_NAME_ONE # First Mender Artifact file name
+  value: artifact_one.mender
+- name: MENDER_ARTIFACT_NAME_TWO # Second Mender Artifact file name in the `MENDER_ARTIFACTS_FOLDER` folder
+  value: artifact_two.mender
+- name: MENDER_RELEASE_NAME_ONE # The name of Mender release for Mender Artifact name `MENDER_ARTIFACT_NAME_ONE`
+  value: auth_keys-1.0
+- name: MENDER_RELEASE_NAME_TWO # The name of Mender release for Mender Artifact name `MENDER_ARTIFACT_NAME_TWO`
+  value: some_config-1.0
```

Comment:
These two Artifacts are really the same. We should use realistic release names, maybe `MENDER_RELEASE_NAME_ONE=hello-mender-one`?

Reply:
Or just `release-1.0` and `release-1.1`?

Reply:
I get that. But names like `auth_keys` are very explicit on something that does not relate to the examples so I was suggesting something more generic.

Following on your point, it could have been like `release-1.0-gateway` and `release-1.0-sensor` where we picture a decentralized system where the same version requires different artifacts on different devices.